src/modules/leaderboard.js

```diff
@@ -0,0 +1,74 @@
+// Create class and add parameterized constructor
+export default class Leaderboard {
+  constructor(user, score) {
+    this.user = user;
+    this.score = score;
+  }
+
+  // Get ul from plain html to dynamically manipulate it
+  static list = document.getElementById('scores-list');
+
+  // Get game ID and cut the extra text with regex matcher (we only need ID)
+  static getGameID = async () => {
+    const namedId = await this.createGame();
+    const reGex = /(?<=Game with ID: ).+(?= )/gi;
+    const id = namedId.result.match(reGex)[0];
+    return id;
+  };
+
+  // Create game with new name and send request to the following URL to load data
+  static createGame = async () => {
+    const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+    const result = await fetch(`${requestURL}/games/`, {
+      method: 'POST',
+      body: JSON.stringify({
+        name: 'Catch the Bugs',
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.json());
+    return result;
+  };
+
+  // Consuming JavaScript Fetch API, sending data to a server with async & await function
+  static add = async (scoreData, id) => {
+    const requestURL = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores/`;
+    document.querySelector('.submit-btn');
+    if (scoreData.user !== '') {
+      const requestOptions = {
+        method: 'POST',
+        headers: {
+          'Content-type': 'application/json; charset=UTF-8',
+        },
+        body: JSON.stringify(scoreData),
+      };
+      await fetch(requestURL, requestOptions);
+    }
+    this.load(id);
+  }
+
+  // Consuming JavaScript Fetch API, receiving data from a server with async & await function
+  static load = async (id) => {
+    const requestURL = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores/`;
+    let scores = '';
+    this.list.innerHTML = '';
+    if (navigator.onLine) {
+      const request = await fetch(requestURL, { method: 'GET' });
+      const { result } = await request.json();
+      if (result.length) {
```

Comment:
- [ ] Great job so far, you're almost there. there is an issue you still to fix. When we click on the `refresh` button, the score list doesn't change its order. Kindly, make sure that you hit well the GET API endpoint to refresh the list. Let's rock, fix this issue.

Reply:
Hello,

After you click refresh button it renders in ascending order (from smallest to largest) so on the top of the list it's the player with the highest score. Is that what you meant by your issue?

Reply:
Hi @SaidRasinlic ,
you're right, It renders in ascending order but just once. when we add other scores it still orders them in ascending order but referencing the requirement, we have to order or refresh the score list just by clicking on the refresh button. you're almost there, that is not really mandatory but try to fix it and then ask for the re-review. Let's rock 😄 🏋️ ... you can do it!

Reply:
Done, I fixed it, I was calling the refresh function on add button because I wanted to refresh it automatically however I fixed it on refreshing manually.